### DISCUSSION TITLE
Error in installing of DNSX

### DISCUSSION THREAD

**Discussion by Eren-0X** [projectdiscovery/dnsx#554] [Q&A]
**#bug**
hello
i got error in installing dnsx at ubuntu
it downloads     github.com/pierrec/lz4 v2.6.1+incompatible
and when it reads it from https://proxy.golang.org/github.com/pierrec/lz4/@v/v2.6.1+incompatible.zip:      it results an error because it has an additional : at the end of link (when i deleted it in browser, it get downloaded)

![bug](https://github.com/projectdiscovery/dnsx/assets/149108455/48e576d7-9c18-49b8-a846-2a6bfbc41c33)

**Comment by olearycrew** [comment]
@Eren-0X I wasn't able to replicate this on a fresh install of Ubuntu when running `go install -v github.com/projectdiscovery/dnsx/cmd/dnsx@latest`

Did you run a different command? What version are you running? You also mentioned a browser - are you using a browser to install or installing via the command line?
